src/paths/distances.c

```diff
@@ -30,14 +30,17 @@
 #include "igraph_vector.h"
 #include "igraph_interface.h"
 #include "igraph_adjlist.h"
+#include "igraph_random.h"
 
 #include "core/interruption.h"
 
 static int igraph_i_eccentricity(const igraph_t *graph,
                                  igraph_vector_t *res,
                                  igraph_vs_t vids,
                                  igraph_neimode_t mode,
-                                 const igraph_adjlist_t *adjlist) {
+                                 const igraph_adjlist_t *adjlist,
+                                 igraph_integer_t *vid_ecc,
```

Comment:
Let's add a comment above the function explaining the meaning of non-obvious parameters. In this case: `vid_ecc`.